ui/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "node-sass-chokidar": "^1.3.3",
```

Comment:
IHMO we should not use `^` since it can bring break changes which may impact the project.